pkg/chains/storage/docdb/docdb.go

```diff
@@ -327,3 +341,58 @@ func getMongoServerURLFromDir(dir string) (string, error) {
 
 	return fileDataNormalized, nil
 }
+
+// getMongoServerUrlFromPath retreives token from the given mount path
+func getMongoServerURLFromPath(path string) (string, error) {
+	fileData, err := os.ReadFile(path)
+	if err != nil {
+		return "", fmt.Errorf("reading file in %q: %w", path, err)
+	}
+
+	// A trailing newline is fairly common in mounted files, so remove it.
+	fileDataNormalized := strings.TrimSuffix(string(fileData), "\n")
```

Comment:
Good idea! Consider even using [string.TrimSpace](https://pkg.go.dev/strings#TrimSpace).

If so, let's do that in `getMongoServerURLFromDir` as well.

(Ok to ignore this comment since it's an already existing pattern.)